extensions/bitwarden/src/create-folder.tsx

```diff
@@ -0,0 +1,76 @@
+import { Form, Action, showHUD, ActionPanel, showToast, Toast } from "@raycast/api";
+import { BitwardenProvider, useBitwarden } from "~/context/bitwarden";
+import RootErrorBoundary from "~/components/RootErrorBoundary";
+import { useState } from "react";
+import { SessionProvider } from "./context/session";
+
+const CreateFolderCommand = () => (
+  <RootErrorBoundary>
+    <BitwardenProvider>
+      <SessionProvider unlock>
+        <CreateFolderComponent />
+      </SessionProvider>
+    </BitwardenProvider>
+  </RootErrorBoundary>
+);
+
+interface FormData {
+  name: string;
+}
+
+function CreateFolderComponent() {
+  const bitwarden = useBitwarden();
+
+  const [nameError, setNameError] = useState<string | undefined>();
```

Comment:
Would you be able to refactor this component to use the [`useForm`](https://developers.raycast.com/utilities/react-hooks/useform) hook? Even though it's just one field, you would be able to remove the `useState`, `validateName` function, and simplify other parts of the code.